pages/linux/add-apt-repository.md

```diff
@@ -1,6 +1,7 @@
 # add-apt-repository
 
 > Manages apt repository definitions.
+> More information: <https://packages.debian.org/buster/software-properties-common>.
```

Comment:
```suggestion
> More information: <https://manned.org/apt-add-repository>.
```
Typically we'd use a man page like <https://manpages.debian.org/latest/software-properties-common/apt-add-repository.1.html>, but <https://manned.org/apt-add-repository> seems to be more extensive here.